### PR TITLE
Account deletion tests

### PR DESCRIPTION
**Pull request by nabil-brn**
### 📝 Description
I've added account deletion tests, from the account itself and from the account lists through the context men

### ❓ Context

- **Impacted projects**: `` LLD
- **Linked resource(s)**: `` https://ledgerhq.atlassian.net/browse/B2CQA-320

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

N/A

### 🚀 Expectations to reach
- 2 first mocked accounts must be deleted.
- Should run successfully on all coins
- No flakiness

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/modals/SettingsAccount/AccountSettingRenderBody.tsx [hunk 1]
```diff
@@ -253,12 +253,12 @@ class AccountSettingRenderBody extends PureComponent<Props, State> {
               danger
               type="button"
               onClick={this.handleOpenRemoveAccountModal}
-              id="account-settings-delete"
+              data-test-id="account-settings-delete-button"
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-desktop/src/renderer/modals/SettingsAccount/AccountSettingRenderBody.tsx:256]: Just double checking nothing used this as an `id` before?

File: apps/ledger-live-desktop/tests/models/AccountPage.ts [hunk 2]
```diff
@@ -43,4 +43,17 @@ export class AccountPage {
...
+
+  async scrollToOperations() {
+    await this.page.locator("id=operation-list").scrollIntoViewIfNeeded();
+  }
+
+  /**
+   * Delete account from account itself
+   */
+  async deleteAccount() {
+    await this.page.locator("data-test-id=account-settings-button").click();
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-desktop/tests/models/AccountPage.ts:55]: Looks fine but why not declare these as reusable locators in the constructor?

File: apps/ledger-live-desktop/tests/models/AccountPage.ts [hunk 3]
```diff
@@ -43,4 +43,17 @@ export class AccountPage {
     await this.stakeBannerButton.scrollIntoViewIfNeeded();
     await this.stakeBannerButton.click();
   }
+
+  async scrollToOperations() {
+    await this.page.locator("id=operation-list").scrollIntoViewIfNeeded();
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-desktop/tests/models/AccountPage.ts:48]: could this be a `data-test-id`?

File: apps/ledger-live-desktop/src/renderer/modals/SettingsAccount/AccountSettingRenderBody.tsx (hunk 1 above)
Comment (nabil-brn) [reply apps/ledger-live-desktop/src/renderer/modals/SettingsAccount/AccountSettingRenderBody.tsx:256]: Nope 👌 thank you I checked again

File: apps/ledger-live-desktop/tests/models/AccountPage.ts (hunk 2 above)
Comment (nabil-brn) [reply apps/ledger-live-desktop/tests/models/AccountPage.ts:55]: You're right, I forgot to rewrite this page object 👍

File: apps/ledger-live-desktop/tests/models/AccountPage.ts (hunk 3 above)
Comment (nabil-brn) [reply apps/ledger-live-desktop/tests/models/AccountPage.ts:48]: I've just moved some code from gre to the right page object, I assumed it wasn't part of my scope for this PR.
We'll change that if needed in a maintenance task later 👍

**Review by ggilchrist-ledger — APPROVED** [review]
LGTM, I didn't go through the screenshots thought as I don't know the context!

**Pull request merged**